jobs/ml_jobs/finance/pyproject.toml

```diff
@@ -0,0 +1,62 @@
+[project]
+name = "finance"
```

Comment:
naming too generic and related to a team (Like calling a ml job "data-science"). Should be reimboursment forecast or something else. Same for folder naming.

Reply:
Keep english (no daf)